libtelnet.c

```diff
@@ -437,7 +436,8 @@ static void _negotiate(telnet_t *telnet, unsigned char telopt) {
 			break;
 		case Q_WANTNO_OP:
 			_set_rfc1143(telnet, telopt, Q_US(q), Q_WANTYES);
-			NEGOTIATE_EVENT(telnet, TELNET_EV_DO, telopt);
+			_send_negotiate(telnet, TELNET_DO, telopt);
```

Comment:
This don't seem to be related to the CMake issues; can you split these out to a separate PR? (Although I believe these are already addressed by another open PR.)

Reply:
https://github.com/seanmiddleditch/libtelnet/pull/57
I don't know why my PR mix into your PR.